Tests/OrdersTests/EncryptedOrdersTests.swift

```diff
@@ -1,116 +1,92 @@
-
-        XCTAssert(FileManager.default.fileExists(atPath: orderFolder.path.appending("/signature")))
-
-        let passJSONData = try String(contentsOfFile: orderFolder.path.appending("/order.json"))
-            .data(using: .utf8)
-        let passJSON = try JSONSerialization.jsonObject(with: passJSONData!) as! [String: Any]
-        XCTAssertEqual(passJSON["authenticationToken"] as? String, order.authenticationToken)
-        try XCTAssertEqual(passJSON["orderIdentifier"] as? String, order.requireID().uuidString)
-
-        let manifestJSONData = try String(
-            contentsOfFile: orderFolder.path.appending("/manifest.json")
-        ).data(using: .utf8)
-        let manifestJSON =
-            try JSONSerialization.jsonObject(with: manifestJSONData!) as! [String: Any]
-        let iconData = try Data(contentsOf: orderFolder.appendingPathComponent("/icon.png"))
-        let iconHash = Array(SHA256.hash(data: iconData)).hex
-        XCTAssertEqual(manifestJSON["icon.png"] as? String, iconHash)
-    }
-
-    func testAPNSClient() async throws {
-        XCTAssertNotNil(app.apns.client(.init(string: "orders")))
-
-        let orderData = OrderData(title: "Test Order")
-        try await orderData.create(on: app.db)
-        let order = try await orderData._$order.get(on: app.db)
-
-        try await ordersService.sendPushNotificationsForOrder(
-            id: order.requireID(), of: order.orderTypeIdentifier, on: app.db)
-
-        let deviceLibraryIdentifier = "abcdefg"
-        let pushToken = "1234567890"
-
-        try await app.test(
-            .POST,
-            "\(ordersURI)push/\(order.orderTypeIdentifier)/\(order.requireID())",
-            headers: ["X-Secret": "foo"],
-            afterResponse: { res async throws in
-                XCTAssertEqual(res.status, .noContent)
-            }
-        )
-
-        try await app.test(
-            .POST,
-            "\(ordersURI)devices/\(deviceLibraryIdentifier)/registrations/\(order.orderTypeIdentifier)/\(order.requireID())",
-            headers: ["Authorization": "AppleOrder \(order.authenticationToken)"],
-            beforeRequest: { req async throws in
-                try req.content.encode(RegistrationDTO(pushToken: pushToken))
-            },
-            afterResponse: { res async throws in
-                XCTAssertEqual(res.status, .created)
-            }
-        )
-
-        try await app.test(
-            .POST,
-            "\(ordersURI)push/\(order.orderTypeIdentifier)/\(order.requireID())",
-            headers: ["X-Secret": "foo"],
-            afterResponse: { res async throws in
-                XCTAssertEqual(res.status, .internalServerError)
-            }
-        )
-
-        // Test `OrderDataMiddleware` update method
-        orderData.title = "Test Order 2"
-        do {
-            try await orderData.update(on: app.db)
-        } catch {}
+    @Test("Test APNS client")
+    func apnsClient() async throws {
```

Comment:
Nit picky, and Not a big deal but since Tim mentioned the names, I also want to mention mentioning "Test" in the name usually isn't very informative. For example this is a swift-testing log:
```
✔ Test "Test APNS client" passed after 23.257 seconds.
```
which has 2 `Test`s one after the other which doesn't add any value.

Reply:
Yeah, you're right, I've fixed them